docs/core/logger.md

```diff
@@ -174,6 +172,9 @@ When debugging in non-production environments, you can instruct Logger to log th
 
     1. Binding your handler method allows your handler to access `this` within the class methods.
 
+Logging incoming events only works with middy or decorator by using `injectLambdaContext`.  
```

Comment:
I think this first sentence is somewhat redundant as it rephrases the one above almost entirely (`you can instruct Logger to log the incoming event with the middleware/decorator`).

I think most of the confusion we have seen reports of is related to the fact that just enabling the env variable is not enough, which is addressed by the next sentence you've added.

Maybe we could just rephrase the initial sentence to be more specific rather than adding another one. What do you think?

Reply:
Simplified wording.